### ISSUE TITLE
Can't Install on Python36 Windows 10

### ISSUE THREAD

**Issue by AgentCosmic** [dschep/ntfy#166]
I get this error when I install using `pip install ntfy`

```
Collecting ntfy
  Using cached ntfy-2.5.1-py2.py3-none-any.whl
Collecting ruamel.yaml (from ntfy)
  Downloading ruamel.yaml-0.15.35-cp36-cp36m-win_amd64.whl (187kB)
    100% |████████████████████████████████| 194kB 3.3MB/s
Requirement already satisfied: requests in c:\python\36\lib\site-packages (from ntfy)
Collecting pypiwin32==219; sys_platform == "win32" (from ntfy)
  Using cached pypiwin32-219.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\Dalton\AppData\Local\Temp\pip-build-756ysjrz\pypiwin32\setup.py", line 121
        print "Building pywin32", pywin32_version
                               ^
    SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Building pywin32", pywin32_version)?
```

**Comment by dschep** [comment]
Hmm. Seems like pypywin32==219 is available for 2.7 and 3.5 while pypiwin32==220 is available for 3.6. I'll have to publish a new release to fix this dep. For now just install on python 2.7 or 3.5.

**Comment by dschep** [comment]
Maybe can switch to plain pywin32 package too, it seems to have wheels now for tons of python versions.
